src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -150,9 +159,18 @@ export function NetworkAlert() {
   const { label, logoUrl, bridge } = CHAIN_INFO[chainId]
   const textColor = TEXT_COLORS[chainId]
 
+  const ALTERNATIVE_LOGO: { [ChainId: number]: string } = {
+    [SupportedChainId.CELO]: celoAlternativeLogo,
+    [SupportedChainId.CELO_ALFAJORES]: celoAlternativeLogo,
+  }
+
   return bridge ? (
     <RootWrapper>
-      <ContentWrapper chainId={chainId} darkMode={darkMode} logoUrl={logoUrl}>
+      <ContentWrapper
+        chainId={chainId}
+        darkMode={darkMode}
+        logoUrl={ALTERNATIVE_LOGO[chainId] ? ALTERNATIVE_LOGO[chainId] : logoUrl}
+      >
```

Comment:
I think I'd prefer this be a `useMemo` calculating `logo` instead of defining a new `ALTERNATIVE_LOGO` object that only matters for celo.

Reply:
Good point! I may remove this all together